packages/blade/package.json

```diff
@@ -66,7 +66,8 @@
     "start:native": "run-p react-native:storybook:start react-native:storybook:android react-native:storybook:ios",
     "start:web": "yarn react",
     "start:all": "run-p start:native start:web",
-    "chromatic": "npx chromatic"
+    "chromatic": "npx chromatic",
+    "npm-publish": "npm publish --tag next --\"@razorpay:registry=https://registry.npmjs.org/\""
```

Comment:
okay since you're explicitly mentioning this here then the `repository` field from package.json is not respected